src/client/godoc_test.go

```diff
@@ -51,7 +51,7 @@ func ExampleAPIClient_DeleteRepo() {
 	if _, err := c.PfsAPIClient.CreateRepo(
 		c.Ctx(),
 		&pfs.CreateRepoRequest{
-			Repo:        NewRepo("test"),
+			Repo:        NewProjectRepo("", "test"),
```

Comment:
If calling NewProjectRepo("", "test") is valid, I think it should just use NewRepo("test")

Reply:
[`NewRepo` is deprecated](https://github.com/pachyderm/pachyderm/pull/8159/files#diff-c9186e1e86b776b3b2a09d2d43691ef924691395938a83f6df7758e3a2250663R27-R28) and this should not (but may be) used at all.